### PR TITLE
Main Page with dummy content

### PR DESCRIPTION
**Pull request by HassanChowdhry**
Screenshot 2023-07-09 at 10 18 13 AM

### CODE REVIEW

File: src/data/data.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
+[
+  {
+    "src": "https://global-uploads.webflow.com/63994dae1033718bee6949ce/649f3f286772ee18775d26f1_mby979eTPSDlRpr7H2MNSly1Sk-p-500.png",
```
Comment (BilalIshtiaque) [inline src/data/data.json:3]: maybe `image` instead ?

File: src/data/data.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    "alt": "Drippi.ai",
```
Comment (BilalIshtiaque) [inline src/data/data.json:4]: `alt` can be hardcoded with some generic picture, perhaps the main HOW logo, no need to get it from the data source, doesnt make sense to define it there

File: src/data/data.json [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    "title": "Drippi.ai",
```
Comment (BilalIshtiaque) [inline src/data/data.json:5]: we also need a link TO the tool, that would technically be the `src`, you can alternatively name it `url` or something

File: src/data/data.json [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+    "description": "A tool to automate personalized Twitter outreach.",
```
Comment (BilalIshtiaque) [inline src/data/data.json:6]: maybe consider more data points based on what sorting options you have ? (sort by recents?)

File: src/layout/Main.tsx [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+        <Typography>
+          <span className='text-grey'>Sort by:</span> Recent  <IconButton aria-label='Sort'><SortIcon /></IconButton>
+        </Typography>
+      </Box>
+
+      <Grid container className='mt-10' rowGap={6}>
+        {DUMMY_DATA.map(({ src, alt, title, description, id }) => (
+          <Article src={src} alt={alt} title={title} description={description} key={id}/>
+        ))}
+      </Grid>
```
Comment (BilalIshtiaque) [inline src/layout/Main.tsx:20]: gotta make it clickable to go to the AI tool website

File: src/layout/Main.tsx [hunk 6]
```diff
@@ -0,0 +1,26 @@
+import { Typography, Box, Grid, IconButton } from '@mui/material';
+import SortIcon from '@mui/icons-material/Sort';
+import Article from '../components/Article';
+import DUMMY_DATA from '../data/data.json';
+
+function Main() {
+  return (
+    <Box className='p-8'>
+      <Box className='flex sm:justify-between sm:flex-row flex-col text-center'>
+        <Typography variant='h6'>Showing 3 of 100 Total Tools.</Typography>
```
Comment (BilalIshtiaque) [inline src/layout/Main.tsx:10]: can u hook it up to the data source (DUMMY_DATA) ?

File: src/data/data.json (hunk 1 above)
Comment (HassanChowdhry) [reply src/data/data.json:3]: ight

File: src/layout/Nav.tsx [hunk 7]
```diff
@@ -1,23 +1,67 @@
...
-        {/* nav links */}
-        <Box>
-          <NavButton to='/'>Home</NavButton>
-          <NavButton to='/about'>About</NavButton>
-          <NavButton to='/add'>Add</NavButton>
-        </Box>
-      </Toolbar>
-    </AppBar>
+          {/* nav links */}
+          {!isMobile
```
Comment (Ebrahim-Merchant) [inline src/layout/Nav.tsx:29]: Nit: This function is getting a little to big maybe you can mobile to Links to a smaller function in the same code. Something like MenuItems or something

File: src/layout/Main.tsx (hunk 6 above)
Comment (HassanChowdhry) [reply src/layout/Main.tsx:10]: u mean by no. of tools in dummy data ryt?

File: src/layout/Main.tsx (hunk 5 above)
Comment (HassanChowdhry) [reply src/layout/Main.tsx:20]: ight

File: src/data/data.json (hunk 4 above)
Comment (HassanChowdhry) [reply src/data/data.json:6]: cant we just sort recents using the ID? And we can sort Alphabetically using the title. maybe add tags for Filter

File: src/layout/Nav.tsx (hunk 7 above)
Comment (HassanChowdhry) [reply src/layout/Nav.tsx:29]: You mean this whole thing in a its own function (same file)?
```
          {!isMobile
            ? <Box>
                <NavButton to='/'>Home</NavButton>
                <NavButton to='/about'>About</NavButton>
                <NavButton to='/add'>Add</NavButton>
              </Box>
            : <IconButton
                size="large"
                edge="start"
                color="secondary"
                aria-label="menu"
                id="basic-button"
                aria-controls={open ? 'basic-menu' : undefined}
                aria-haspopup="true"
                aria-expanded={open ? 'true' : undefined}
                onClick={handleClick}
              >
                <MenuIcon />
              </IconButton>
          }
```

**Review by BilalIshtiaque — APPROVED** [review]

**Pull request merged**